### DISCUSSION TITLE
Why doesn't the node installed by brew support mojave. Downloaded from nodejs official website can be installed in mojave

### DISCUSSION THREAD

**Discussion by airstarlabs** [Homebrew/discussions#2442] [Everyday usage]
Why doesn't the node installed by brew support mojave. Downloaded from nodejs official website can be installed in mojave

**Comment by carlocab** [comment]
> See https://brew.sh/2021/10/25/homebrew-3.3.0/. We only build bottles for the three latest versions of macOS. You should still be able to build Node from source if you're running Mojave. We're happy to review pull requests that fix Mojave builds if that's not the case.

https://github.com/Homebrew/homebrew-core/commit/bd4ab38b84f51fff8b5e34b4ae401afc5cefeb66#commitcomment-60157695

**Reply by airstarlabs** [reply]
Is there a way to ignore the environment check installation

**Reply by airstarlabs** [reply]
The latest three systems are not the best systems for my 2019 mac pro. I was downgraded yesterday after the upgrade.

**Reply by carlocab** [reply]
Note that even though upstream Node says that their builds are compatible with Mojave, that doesn't mean ours are, because the two builds are configured differently. There's no guarantee that the Node bottles (or any of its dependencies) will not make system calls that are only available on the version that they were built.

That said, if you truly wish to ignore the OS version check, you can fetch the bottle you want manually with `brew fetch` and install it with `brew install /path/to/bottle`.

**Comment by naryfa** [comment]
I guess I'm screwed. I can't go higher than Mojave :(

**Reply by gromgit** [reply]
Some of Node's dependencies are still available as Mojave bottles. As for the rest, I just compiled Node itself and all its remaining non-bottled dependencies from source on Mojave, on a dual-core VM that's maybe 10% as powerful as your Mac Pro, in just a couple of hours. I doubt you're as screwed as you think you are. :grin: 

Try `brew install -s node` and see if it gets you to where you need to be.
